english/dance/dance.md

```diff
@@ -9,6 +9,8 @@ The cake is ready to bake.
 
 But there are too many cooks!
 
+Or so you think. You're not sure because you are self-aware of your tendency to be easily swayed by catchy rock songs of the 80s and Foreigner - Double Vision was just on the radio.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.